### ISSUE TITLE
gradle output shows signing passwords in cleartext

### ISSUE THREAD

**Issue by Syrou** [fastlane/fastlane#4385]
### New Issue Checklist

- [ x] Updated fastlane to the latest version
- [ x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description
When running:

```
gradle(
            task: 'assemble',
            flavor: 'live',
            build_type: 'Release',
            properties: {
                'versionCode' => ENV["CI_BUILD_ID"],
                'versionName' => last_git_tag.to_s,
                'android.injected.signing.store.file'=>"#{store_file}",
                'android.injected.signing.store.password'=>"#{store_password}",
                'android.injected.signing.key.alias'=>"#{key_alias}",
                'android.injected.signing.key.password'=>"#{key_password}"
            },
            flags: "--refresh-dependencies"
        )
```

the variables will be printed in cleartext, this is specificly bad in CI enviroments as it's also saved in the build logs. There needs to be a way to make the gradle command silent, if there already is not?

**Comment by vpolouchkine** [comment]
What if you pass `--quiet` to the `flags` argument?

**Comment by Syrou** [comment]
While this will hide the output from gradle, it won't hide the input shell command, which is where the problem is :)

Currently this command is being issued by Action.sh(command) in the [https://github.com/fastlane/fastlane/blob/c74aced9fe5d7c1f948dd16bd200a2167ff5534f/fastlane/lib/fastlane/helper/gradle_helper.rb](url)

Possibly, this should be updated to become Actions.sh(command, log: optionable)

**Comment by mfurtak** [comment]
@Syrou Thanks for reporting this 👍 

We're treating this as a high priority problem, so we'll be in touch when we have a solution.

**Comment by mfurtak** [comment]
We've merged a solution in https://github.com/fastlane/fastlane/pull/4393, and expect to release `fastlane` tomorrow to make it available for you! 🚀

**Comment by Syrou** [comment]
Awesome work! 👍

**Comment by mfurtak** [comment]
Hi @Syrou 👋 

This shipped as part of [`fastlane` 1.83.0](https://github.com/fastlane/fastlane/releases/tag/1.83.0) 🚀 

Once you update you should be able to better control `gradle` action output by using the `print_command` and `print_command_output` options!
